test/index.ts

```diff
@@ -295,60 +309,43 @@ describe('PubSub', () => {
       name: 'subscription-name',
     };
 
-    beforeEach(() => {
-      // tslint:disable-next-line no-any
-      (Subscription as any).formatMetadata_ = metadata => {
-        return Object.assign({}, metadata);
-      };
-    });
 
-    it('should throw if no Topic is provided', () => {
```

Comment:
looks like these two tests got deleted

Reply:
Topic is a required parameter
https://github.com/googleapis/nodejs-pubsub/blob/master/src/index.ts#L339

Reply:
Yes, TypeScript will get angry.  However, not all of our users are using TypeScript :) You need to cast pubsub as an `any` or cast `createSubsscription` as `Function` below.  Lets keep these tests.